x-pack/plugins/observability_solution/observability/public/pages/alerts/alerts.tsx

```diff
@@ -177,8 +180,8 @@ function InternalAlertsPage() {
     try {
       const response = await loadRuleAggregations({
         http,
-        typesFilter: filteredRuleTypes,
-        filterConsumers: observabilityAlertFeatureIds,
+        ruleTypeIds: filteredRuleTypes,
+        consumers: observabilityAlertFeatureIds,
```

Comment:
nit: maybe we can later clean up observabilityAlertFeatureIds to something like observabilityAlertConsumers. The current naming is confusing.